packages/cli/src/config.ts

```diff
@@ -217,15 +217,20 @@ export const gqlessConfigPromise: Promise<{
                 const { format } = (await import('./prettier')).formatPrettier({
                   parser: 'typescript',
                 });
+
+                const config: GQlessConfig = { ...defaultConfig };
+                delete config.preImport;
+                delete config.enumsAsStrings;
```

Comment:
can you share why are you making this change?

Reply:
default config doesn't need these, advanced usage only